templates/web/common/includes/display_possible_improvement_description.tt.html

```diff
@@ -0,0 +1,21 @@
+<!-- start templates/[% template.name %] -->
+
+[% IF product_ref_improvements_data.defined && product_ref_improvements_data_tagid.defined %]
```

Comment:
this check should be in the Perl code, and we should not call the template if we don't have improvements data defined